google-ads.yaml

```diff
@@ -7,6 +7,16 @@
 ###############################################################################
 developer_token: INSERT_DEVELOPER_TOKEN_HERE
 
+# Use proto plus
+###############################################################################
+# This parameter specifies whether the client library should return           #
+# proto-plus messages or protobuf messages. This value should be explicitly   #
+# set to either "True" or "False", For more information on the                #
+# differences between these two types, see our Protobuf Messages guide:       #
+# https://developers.google.com/google-ads/api/docs/client-libs/python/protobuf-messages #
```

Comment:
Nit: expand the right edge of the table so it's as wide as the link (unless that's more anti-pep-8?)

Reply:
Since this isn't a Python file I don't think it is subject to pep-8, so I updated it